### PR TITLE
BI-8: linhhoang16 - Week 5: Develop test script for below functions o…

### PR DESCRIPTION
**Pull request by linhhoang16**
## Exercise
 * [Exercise- Week 5](https://github.com/NT-Line-team/Behave_test/issues/8)

 ## Overview
 Develop test script for below function of Opencart web application:
 1. Login
 Execute a successful login with admin user.
 2. Register
 Register a new user.
 3. Forgot password
 Test the forgot password function to make sure that the user receive an inform message after submitting the request.
 
 ## Verification Steps:
 Pls run below commands:
 1. `cd /Behave_test/exercises/linhhoang16`
 2. To execute login test script pls run `python ./login.py`
 3. To execute register test script pls run `python ./register.py`
 4. To execute forgot password test script pls run `python ./forgot_password.py`

### CODE REVIEW

**Comment by duynguyenx** [comment]
1. The Pull request description: It'd be better if you can give some headline for each section, pls see example below
 

> ## Exercise
> * [Exercise- Week 5](https://github.com/NT-Line-team/Behave_test/issues/8)
> 
> ## Overview
> Develop test script for below function of Opencart web application:
> 1. Login
> Execute a successful login with admin user.
> 2. Register
> Register a new user.
> 3. Forgot password
> Test the forgot password function to make sure that the user receive an inform message after submitting the request.
> 
> ## Verification Steps:
> Pls run below commands:
> 1. `cd /Behave_test/exercises/tamhuynh`
> 2. To execute login test script pls run `python ./login.py`
> 3. To execute register test script pls run `python ./register.py`
> 4. To execute forgot password test script pls run `python ./forgot_password.py`


2. Login and Forgot password is 2 test cases, so we should have 2 test script.

File: exercises/linhhoang16/login_forgotpassword.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from selenium import webdriver
+
+driver = webdriver.Chrome()
+baseurl = "http://localhost:8080/shop/"
+driver.get(baseurl)
+driver.implicitly_wait(2)
```
Comment (duynguyenx) [inline exercises/linhhoang16/login_forgotpassword.py]: We should use implicit_wait 1 time only and it should be after maximize the windows. Pls help to remove others.

File: exercises/linhhoang16/login_forgotpassword.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+driver.maximize_window()
+
+my_account = driver.find_element_by_link_text("My Account").click()
```
Comment (duynguyenx) [inline exercises/linhhoang16/login_forgotpassword.py]: Do not need to save to param `my_account` here. This line of code aims to click on My Account link so it should be `driver.find_element_by_link_text("My Account").click()`

Pls also correct others

**Comment by linhhoang16** [comment]
Hi anh @duynguyenx ,

I have updated the code as per your comments. Please help review.

Thanks in advance,

Linh

**Comment by duynguyenx** [comment]
@linhhoang16  pls check other comments in Conversation tab.

**Comment by linhhoang16** [comment]
Hi anh @duynguyenx ,

I have done for other comments:
1. Update PR description
2. Separated login_forgotpassword.py to 2 files

Please hep me review

Thanks,
Linh

**Comment by duynguyenx** [comment]
LGTM

**Pull request merged**